### PR TITLE
[WIP] docs: add business funnel demo and related docs

### PR DESCRIPTION
**Pull request by LofiSu**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a Demo add business funnel demo and related docs

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [ ] Site / documentation update
- [ √] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related PR link

https://github.com/VisActor/VChart/issues/2657

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: docs/assets/examples/en/funnel-chart/business-funnel-template.md [hunk 1]
```diff
@@ -0,0 +1,571 @@
+---
+category: examples
+group: funnel chart
+title: Business Funnel Chart Template
+keywords: funnelChart,composition,trend,custom,template,customMark
+cover: /vchart/preview/funnel-business-template-1.1.png
```
Comment (xiaoluoHe) [inline docs/assets/examples/en/funnel-chart/business-funnel-template.md:6]: 1.1 和 1.0 的命名区别有点困惑
参照其他 demo，规范一下：`图片名_上线官网的版本号.png`

File: docs/assets/examples/zh/funnel-chart/business-funnel-template.md [hunk 2]
```diff
@@ -0,0 +1,573 @@
...
+    return '';
+  }
+  const currentValue = data[curIndex].value;
+  const nextValue = data[curIndex + 1]?.value ?? 0;
+
+  return `${Math.trunc(currentValue - nextValue)}`;
+}
+
+//默认渲染次数为0
+let renderCount = 0;
```
Comment (xiaoluoHe) [inline docs/assets/examples/zh/funnel-chart/business-funnel-template.md:41]: 什么作用？

File: docs/assets/examples/zh/funnel-chart/business-funnel-template.md [hunk 3]
```diff
@@ -0,0 +1,573 @@
...
+            size: 18,
+            scaleX: 0.8,
+            symbolType: 'arrow2Right',
+            cornerRadius: 5,
+            fill: 'rgb(200,200,200)'
+          }
+        },
+        //箭头尾巴
+        {
+          type: 'symbol',
```
Comment (xiaoluoHe) [inline docs/assets/examples/zh/funnel-chart/business-funnel-template.md:267]: 有点整复杂了，一个 symbol svg 就可以，不要用两个形状拼

File: docs/assets/examples/en/funnel-chart/business-funnel-template.md [hunk 4]
```diff
@@ -0,0 +1,571 @@
...
+        {
+          type: 'text',
+          dataIndex: 0,
+          style: {
+            text: (datum, ctx, params, dataView) => {
+              const data = dataView.latestData;
+              if (!data) return '';
+
+              // Define the calculation relationship for each label
+              const labelMap = {
```
Comment (xiaoluoHe) [inline docs/assets/examples/en/funnel-chart/business-funnel-template.md:468]: 常量并且不止一处用到，不要写在这个函数里，放到外边去

File: docs/assets/examples/en/funnel-chart/business-funnel-template.md (hunk 1 above)
Comment (LofiSu) [reply docs/assets/examples/en/funnel-chart/business-funnel-template.md:6]: 好滴

File: docs/assets/examples/zh/funnel-chart/business-funnel-template.md (hunk 2 above)
Comment (LofiSu) [reply docs/assets/examples/zh/funnel-chart/business-funnel-template.md:41]: 记录渲染次数，下面有用到

File: docs/assets/examples/zh/funnel-chart/business-funnel-template.md (hunk 3 above)
Comment (LofiSu) [reply docs/assets/examples/zh/funnel-chart/business-funnel-template.md:267]: 好滴

**Pull request closed without merging**